.github/workflows/dockerhub-release-orioledb.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - s3-config
```

Comment:
remove if ready for merge

Reply:
Please don't merge. It's blocked on orioledb testing as mentioned in description.